segmentation/include/pcl/segmentation/grabcut_segmentation.h

```diff
@@ -286,6 +286,8 @@ namespace pcl
         uint32_t count_;
         /// small value to add to covariance matrix diagonal to avoid singular values
         float epsilon_;
+        public:
+          EIGEN_MAKE_ALIGNED_OPERATOR_NEW
```

Comment:
Can we just put this under already existing `public:` section?

Reply:
Sure, I will fix that!